dispatcher/MongoConnect.py

```diff
@@ -209,7 +212,13 @@ def aggregate_status(self):
                 statuses[doc['host']] = status
                 phys_stat[phys_det].append(status)
 
-            for doc in self.latest_status[detector]['controller'].values():
+            for n, doc in self.latest_status[detector]['controller'].items():
+                if doc is None:
+                    self.logger.debug(f'{n} seems to have been offline for a few days')
+                    modes.append('none')
+                    run_nums.append(-1)
+                    status_list.append(DAQ_STATUS.UNKNOWN)
```

Comment:
the logic is too complex for me to see why the `status_list` would already be declared here

Reply:
That's because it isn't.